### PR TITLE
Added in-toto Favicon

### PR DESCRIPTION
**Pull request by DarikshaAnsari**
- Fixes :- #24

### CODE REVIEW

**Comment by DarikshaAnsari** [comment]
@chalin PTAL!!

**Review by lukpueh — COMMENTED** [review]
Looks good. Any reason, you didn't include the standard 16x16 favicon? 

See opentelemtry for reference:
https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/partials/favicons.html
https://github.com/open-telemetry/opentelemetry.io/tree/main/static/favicons

**Comment by DarikshaAnsari** [comment]
> Looks good. Any reason, you didn't include the standard 16x16 favicon?
> 
> See opentelemtry for reference: https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/partials/favicons.html https://github.com/open-telemetry/opentelemetry.io/tree/main/static/favicons

I'm not sure. I used [this icon generator](http://cthedot.de/icongen), which gave me a zip folder containing only specific sizes. If you'd like, I can include all the sizes with reference to OTEL.

**Comment by lukpueh** [comment]
> I'm not sure. I used [this icon generator](http://cthedot.de/icongen), which gave me a zip folder containing only specific sizes. If you'd like, I can include all the sizes with reference to OTEL.

Not sure, if that's necessary. @chalin?

I suggest you let cthedot generate the default icons (the ones that are already selected):
```
favicon.ico
16x16 only, add to website root but do not add to HTML

PWA Progressive Web Apps
192x192 512x512 + maskable 192x192 512x512

apple touch icon
180x180
```

Also, in case you didn't already do that, you can use this svg as base image: https://github.com/cncf/artwork/blob/main/projects/in-toto/icon/color/in-toto-icon-color.svg

**Comment by DarikshaAnsari** [comment]
see i have done this but this is not generating the desired output just check once.

**Comment by lukpueh** [comment]
Weird, when I do it, I get this zip:

[icongen.zip](https://github.com/user-attachments/files/16760411/icongen.zip)

**Comment by DarikshaAnsari** [comment]
Now I have added all the files of the zip that you've provided.

File: static/favicons/_head.html
Comment (chalin) [inline static/favicons/_head.html:1]: Drop this file, unless you know that it is being used, and if so explain why you believe so.

File: layouts/partials/favicons.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{{/* Docsy-delta */ -}}
```
Comment (chalin) [inline layouts/partials/favicons.html]: Let's keep a note about how these were generated:

```suggestion
{{/* Docsy-delta

Icons generated by https://cthedot.de/icongen
from https://github.com/cncf/artwork/blob/main/projects/in-toto/icon/color/in-toto-icon-color.svg
*/ -}}
```

File: layouts/partials/favicons.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+<link rel="shortcut icon" href="{{ "favicons/favicon.ico" | relURL }}" >
+<link rel="apple-touch-icon" href="{{ "favicons/apple-touch-icon.png" | relURL }}" sizes="180x180">
+
+<!-- Android Icons -->
```
Comment (chalin) [inline layouts/partials/favicons.html]:
```suggestion
```

File: layouts/partials/favicons.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<link rel="icon" type="image/png" href="{{ "favicons/android-chrome-192x192.png" | relURL }}" sizes="192x192">
+<link rel="icon" type="image/png" href="{{ "favicons/android-chrome-512x512.png" | relURL }}" sizes="512x512">
+
+<!-- Android Maskable Icons -->
```
Comment (chalin) [inline layouts/partials/favicons.html]:
```suggestion
```

**Review by chalin — CHANGES_REQUESTED** [review]
Nice to see the favicon ✨. The set of icon files included seems fine.

See inline comments for a file to be dropped.

**Review by lukpueh — APPROVED** [review]

**Review by chalin — APPROVED** [review]

**Pull request merged**